src/services/line-stat-getter.js

```diff
@@ -102,8 +102,8 @@ function LineStatFinder(model, params, opts) {
     const groupByFieldName = populateGroupByField
       ? params.group_by_field.replace(':', '.') : params.group_by_field;
 
-    return new P((resolve, reject) => {
-      const jsonQuery = queryBuilder.getQueryWithFiltersAndJoins(null, true);
+    return new P(async (resolve, reject) => {
+      const jsonQuery = await queryBuilder.getQueryWithFiltersAndJoins(null);
```

Comment:
line chart is long

Reply:
Same issue. If you do it on a 2 collections this PR doesn't fix bad usage ^^

Reply:
Ok, it would be nice to have the scope, since I didn't know I give you out of scope remarks.